Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/GenericComboBoxCtrl.h

```diff
@@ -92,6 +92,7 @@ namespace AzToolsFramework
         void addElements(const AZStd::vector<AZStd::pair<T, AZStd::string>>& genericValues);
         void addElement(const AZStd::pair<T, AZStd::string>& genericValue);
         void setElements(const AZStd::vector<AZStd::pair<T, AZStd::string>>& genericValues);
+        void clearElements();
```

Comment:
Nit: We prefer upper camel-case for our methods.
```suggestion
        void ClearElements();
```

Reply:
All the rest of the methods in this class use the Qt-style `lowerCamelCase`, I'd think the rule to follow the style of the rest of the existing code would apply here.